portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,214 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Edson's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+     <link rel="stylesheet" href="style.css">
+    <title>Edson Casimiro</title>
+    <!--adding google fonts and fontawesome-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css">
+</head>
+<body>
+
+    <nav>
+        <div class="logo">
+            Edson Casimiro
+        </div>
+        <div class="hamburger">
+            <div class="line1"></div>
```

Comment:
Google's HTML style guide recommends indenting by 2 spaces, not 4: https://google.github.io/styleguide/htmlcssguide.html#Indentation

Like we talked about in the first meeting, I'm happy to leave it up to you how strictly you want to follow the style guide, but I'll mention it when I notice things.

Reply:
Thanks so much, i thought its like some script languages... I will change it, thanks.